backend/prompt_studio/prompt_studio_output_manager/output_manager_helper.py

```diff
@@ -144,3 +145,37 @@ def fetch_default_llm_profile(tool: CustomTool) -> ProfileManager:
             return ProfileManager.get_default_llm_profile(tool=tool)
         except DefaultProfileError:
             raise DefaultProfileError("Default ProfileManager does not exist.")
+
+    @staticmethod
+    def fetch_default_response(
+        tool_studio_prompts: list[ToolStudioPrompt], document_manager_id: str
+    ) -> dict[str, Any]:
+        # Initialize the result dictionary
+        result: dict[str, Any] = {}
+        # Iterate over ToolStudioPrompt records
+        for tool_prompt in tool_studio_prompts:
+            prompt_id = str(tool_prompt.prompt_id)
+            profile_manager_id = str(tool_prompt.profile_manager.profile_id)
+
+            # If profile_manager is not set, skip this record
+            if not profile_manager_id:
+                result[tool_prompt.prompt_key] = ""
+                continue
+
+            try:
+                queryset = PromptStudioOutputManager.objects.filter(
+                    prompt_id=prompt_id,
+                    profile_manager=profile_manager_id,
+                    is_single_pass_extract=False,
+                    document_manager_id=document_manager_id,
+                )
+
+                if not queryset.exists():
+                    result[tool_prompt.prompt_key] = ""
+                    continue
+
+                for output in queryset:
+                    result[tool_prompt.prompt_key] = output.output
+            except ObjectDoesNotExist:
```

Comment:
@chandrasekharan-zipstack Yes, even I had this doubt. @jagadeeswaran-zipstack was there any specific use case to add this logic? Also, I doubt `MultipleObjectsReturned` will be raised here, in case of multiple objects fetched. Should we use `get` instead of `filter`?

Reply:
We dont need that additional check there. Ill address that in my PR

Reply:
I don't have full context on the structure of objects here, but here's the [docs for get()](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#get) - please go through and validate if we're doing what's intended @harini-venkataraman @jagadeeswaran-zipstack